### ISSUE TITLE
`git lfs fetch <origin> --all` gets stuck at `0B` and doesn't advance (but doesn't throw error either)

### ISSUE THREAD

**Issue by JonasT** [git-lfs/git-lfs#1960]
I'm just trying to fetch all LFS objects from a remote repo of mine, and it doesn't work:

```
jonas@cyberman:~/Develop/cicp$ git lfs fetch wobbleninja --all
Scanning for all objects ever referenced...
✔ 25 objects found                                                                                                
Fetching objects...
Git LFS: (0 of 23 files) 0 B / 17.59 MB                                                                          UGit LFS: (0 of 23 files) 0 B / 17.59 MB
```
It just gets stuck there and nothing happens. I can CTRL+C and retry, the same thing. The server I'm pulling from (which is my own rented server instance) is otherwise perfectly reachable, and LFS has usually worked just fine.

This is the origin I'm pulling from (which is currently a public repo):
```
[remote "wobbleninja"]
    url = https://gitlab.wobble.ninja/ChiCraft/cicp
    fetch = +refs/heads/*:refs/remotes/origin/*
```

Since I don't get any error or other info, I'm not sure what to do...

**Comment by ttaylorr** [comment]
Hi @JonasT, I fixed a similar bug this week. Would you mind trying this again on `master`? If that doesn't work, I have a few other ideas of what might be causing your problem.

**Comment by evitolins** [comment]
A few of us on our team are having similar issues recently with freshly cloned repo.  It seems that some older versions of LFS seem have issues prompting for username & password properly.

I found that two things that helped was:

1. Make sure you're using the latest LFS version
2. make sure the remote URL includes the username (ie. https://username@repo.com/reponame)

Still even with this, the behavior hasn't been completely predictable.  Our team is on Mac, Windows and Linux and still sorting out all the quirks.

This issue in one form or another seems pretty common, but still unable to track down a solid documented solution.

**Comment by ttaylorr** [comment]
> 1. Make sure you're using the latest LFS version

#1955 (which will be released as a part of v2.0.0) should fix the bug where the progress constantly shows `0 B / xxx B`.

> 2. make sure the remote URL includes the username (ie. https://username@repo.com/reponame)

https://github.com/git-lfs/git-lfs/pull/1886 (also scheduled to be released in v2.0.0) should fix this.

**Comment by ttaylorr** [comment]
Hi @evitolins a number of bugs have been fixed pertaining to this issue since you originally opened it. Can you try this again with the latest version of Git and Git LFS installed and let me know if it remains an issue. If so, please do not hesitate to comment here and I'd be happy to re-open the issue.
